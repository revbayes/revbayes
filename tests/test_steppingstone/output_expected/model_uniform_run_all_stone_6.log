Iteration	Posterior	Likelihood	Prior	alpha	bl[1]	bl[2]	bl[3]	bl[4]	bl[5]	bl[6]	bl[7]	bl[8]	bl[9]	bl[10]	bl[11]	bl[12]	bl[13]	bl[14]	bl[15]	bl[16]	bl[17]	bl[18]	bl[19]	bl[20]	bl[21]	bl[22]	bl[23]	bl[24]	bl[25]	bl[26]	bl[27]	bl[28]	bl[29]	bl[30]	bl[31]	bl[32]	bl[33]	bl[34]	bl[35]	bl[36]	bl[37]	bl[38]	bl[39]	bl[40]	bl[41]	bl[42]	bl[43]	er[1]	er[2]	er[3]	er[4]	er[5]	er[6]	gamma_rates[1]	gamma_rates[2]	gamma_rates[3]	gamma_rates[4]	pi[1]	pi[2]	pi[3]	pi[4]	pinvar	TL
0	-22533.69	-22530.29	-3.401854	0.8257322	0.0317055	0.115133	0.157467	0.148591	0.030181	0.110097	0.226939	0.336547	0.11762	0.0174008	0.248753	0.0932389	0.254538	0.266509	0.0883007	0.127795	0.0159498	0.0220965	0.0764404	0.0445924	0.22192	0.118824	0.175728	0.0232426	0.0676874	0.00456583	0.0490604	0.032471	0.0895864	0.145716	0.022725	0.152619	0.106344	0.0462462	0.0187607	0.101796	0.0516792	0.0685291	0.181843	0.0470804	0.0437589	0.0344063	0.648234	0.0421139	0.572545	0.0334128	0.0389201	0.248106	0.0649019	0.101021	0.417291	0.963838	2.51785	0.3225	0.384619	0.0517041	0.241176	0.3958611	4.982718
10	-22880.7	-22876.61	-4.090426	2.315767	0.0236994	0.165495	0.232076	0.112579	0.0213022	0.103717	0.0725022	0.281425	0.0794979	0.0772943	0.249196	0.0918669	0.228757	0.278293	0.0680775	0.227345	0.00627596	0.0949554	0.0385593	0.105914	0.08646	0.101327	0.334769	0.0187632	0.0998355	0.00678071	0.133048	0.0475754	0.0383569	0.135107	0.00639375	0.0997813	0.0705965	0.190522	0.0352056	0.0614543	0.152105	0.0918398	0.0753433	0.0373132	0.134949	0.0148866	0.5275	0.0615172	0.313784	0.0329454	0.0711826	0.492698	0.027873	0.328264	0.684876	1.0762	1.91066	0.326326	0.317737	0.0840935	0.271843	0.4504409	5.05874
