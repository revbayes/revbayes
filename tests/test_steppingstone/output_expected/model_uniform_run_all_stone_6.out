state	power	likelihood
4	0.01024	-22520.3
5	0.01024	-22453
6	0.01024	-22683.9
7	0.01024	-23220.9
8	0.01024	-22865.1
9	0.01024	-22799.8
10	0.01024	-22876.6
